### ISSUE TITLE
Version number for the standard?

### ISSUE THREAD

**Issue by jsy1001** [emmt/OI-Interface#20]
I propose applying a version number to the interface specification defined in "Interface to Image Reconstruction." The image reconstruction codes can then advertise themselves as being compatible with a specific version of the interface.

We could follow the [semver](https://semver.org/) conventions for incrementing the version number.

**Comment by FerreolS** [comment]
Good idea, we all current issue will be settled, I propose to make a release through github and tag with a version number. Should this version be the v1.0 (`Version 1.0.0 defines the public API` and `Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable` in [semver](https://semver.org/) )?

we can also provide a doi through zenodo
